### PR TITLE
Rename mxnet-model-server to multi-model-server

### PR DESCRIPTION
**Pull request by alexwong**
WIP, may be a two part PR. 

## Testing done:

**To run CI tests on your changes refer [README.md](https://github.com/awslabs/mxnet-model-server/blob/master/ci/README.md)**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: benchmarks/README.md [hunk 1]
```diff
@@ -26,10 +26,10 @@ The benchmarking script requires the following to run:
 
 ## Models
 
-The pre-loaded models for the benchmark can be mostly found in the [MMS model zoo](https://github.com/awslabs/mxnet-model-server/blob/master/docs/model_zoo.md).  We currently support the following:
-- [resnet: ResNet-18 (Default)](https://github.com/awslabs/mxnet-model-server/blob/master/docs/model_zoo.md#resnet-18)
-- [squeezenet: SqueezeNet V1.1](https://github.com/awslabs/mxnet-model-server/blob/master/docs/model_zoo.md#squeezenet_v1.1)
-- [lstm: lstm-ptb](https://github.com/awslabs/mxnet-model-server/blob/master/docs/model_zoo.md#lstm-ptb)
+The pre-loaded models for the benchmark can be mostly found in the [MMS model zoo](https://github.com/awslabs/multi-model-server/blob/master/docs/model_zoo.md).  We currently support the following:
```
Comment (vdantu) [inline benchmarks/README.md]: These links wont work until the project name is changed.

File: docker/advanced_settings.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ Save the file.
 When you run the following command, the `-v` argument and path values of `/tmp/models/:/models` will map the `models` folder you created (assuming it was in ) with a folder inside the Docker container. MMS will then be able to use the local model file.
 
 ```bash
-nvidia-docker run -itd --name mms -p 80:8080  -p 8081:8081 -v /tmp/models/:/models awsdeeplearningteam/mxnet-model-server:latest-gpu mxnet-model-server --start --mms-config /models/config.properties --models squeezenet=https://s3.amazonaws.com/model-server/models/squeezenet_v1.1/squeezenet_v1.1.model
+nvidia-docker run -itd --name mms -p 80:8080  -p 8081:8081 -v /tmp/models/:/models awsdeeplearningteam/mxnet-model-server:latest-gpu multi-model-server --start --mms-config /models/config.properties --models squeezenet=https://s3.amazonaws.com/model-server/models/squeezenet_v1.1/squeezenet_v1.1.model
```
Comment (vdantu) [inline docker/advanced_settings.md]: Container names should change. Make it `awsdeeplearningteam/multi-model-server`

File: benchmarks/README.md (hunk 1 above)
Comment (alexwong) [reply benchmarks/README.md]: I purposefully didn't just replace everything as not everything has actually been updated (docker containers, s3 buckets, etc.). Above is fine, will change project name once merged, unless you think we should do it before.

File: docker/advanced_settings.md (hunk 2 above)
Comment (alexwong) [reply docker/advanced_settings.md]: The actual containers hosted on dockerhub haven't been updated is why I didn't update this.

**Comment by vdantu** [comment]
I don't see issues with this PR. I would recommend doing container and pypi package changes soon as well. Which means you would need to change the container references here. 

Do note a TODO to go over all the references of mxnet-model-server in the repo again.

File: setup.py [hunk 3]
```diff
@@ -165,8 +165,8 @@ def run(self):
         },
         entry_points={
             'console_scripts': [
-                'mxnet-model-server=mms.model_server:start',
-                'mxnet-model-export=mms.export_model:main'
+                'multi-model-server=mms.model_server:start',
+                'multi-model-export=mms.export_model:main'
```
Comment (vdantu) [inline setup.py]: `mxnet-model-export`  isn't required anymore. mxnet-model-export was the original name of model-archiver. We have it here for backeward compatibility.

**Comment by alexwong** [comment]
Manually checked code build and it succeeded. Merging in.

**Pull request merged**